docs/usage.md

```diff
@@ -36,6 +36,7 @@ The full list of supported environment variables is described below:
 
 |Env Var|Description|Default Value|
 |-----------|-------------|------|
+|`KUBECHECKS_ADDITIONAL_NAMESPACES`|Additional namespaces other than the ArgoCDNamespace to monitor for applications.|`[]`|
```

Comment:
IMO, I think we should have one variable for configuring namespace, `KUBECHECKS_MONITOR_APPS_NAMESPACES`? this can default to `argocd` namespace, but when it's empty it monitors all namespaces. What do you think @djeebus ?

Reply:
I'd like to mimic argocd's behavior in this case. According to my reading of [the docs](https://argo-cd.readthedocs.io/en/stable/operator-manual/app-any-namespace/), by default argocd will *only* monitor the argocd installation namespace (which defaults to `argocd`, but can be changed). You can add `--application-namespaces=a,b,c`, and it will monitor those namespaces additonally 

... although after rereading the docs, it's not clear to me if this is "in addition to" the installation namespace, or "instead of". I suppose some testing is in order. If it's "instead of", then I agree with you, we should skip the argocd installation namespace when this is set. Mind reading the docs linked above and let me know if you agree?

Reply:
The docs do point out that those strings (as parsed by argocd) can be:

- A raw namespace: `namespace-a`
- A glob: `namespace-*`
- A regex: `/^namespace-(\d+)$/`
 
Think we need to support those in order to resolve this issue?

Reply:
I went through the docs and it's not entirely clear if these namespaces are in addition to or including the argocd namespace. Although, I'm thinking it's the former. 

In that case, I'm happy with having `KUBECHECKS_ARGOCD_ADDITIONAL_NAMESPACES`. We just need to cater for the `*`, which is when all namespaces are included.

Reply:
@MeNsaaH @djeebus The recent changes I pushed bring us closer to how ArgoCD handles the informers. Rather than checking for each namespace allowed, we now watch for resources in all namespaces and filter for the allowed ones only. This list of additional namespaces supports `*` or any of the other glob patterns.